pyccel/codegen/printing/fcode.py

```diff
@@ -1408,32 +1402,65 @@ def _print_FunctionDef(self, expr):
             sig = 'elemental {}'.format(sig)
 
         arg_code  = ', '.join(self._print(i) for i in chain( expr.arguments, out_args ))
+
+        imports = ''.join(self._print(i) for i in expr.imports)
+        prelude = ''.join(self._print(i) for i in args_decs.values())
+        parts = ["{}({}) {}\n".format(sig, arg_code, func_end,),
```

Comment:
I suggest to remove `\n` from this string, in order to make the printed interface a little more compact.

Reply:
This function, is used in printing both function definition and interfaces, should i return a dictionary and then decide how to print in the calling method ?

Reply:
Sorry, I thought this was only used for the interface (given its name). Yes. that's a good idea.